team/User-testing.md

```diff
@@ -0,0 +1,69 @@
+# User Testing
+
+User testing is when we schedule time with one person, and observe them using an application or prototype.
+
+## When to do user testing?
+* As part of design thinking, develop protypes and test them with as soon as possible
+* After significant revisions to a design, you can schedule additional testing
+* Before starting development on a design or mockup
+* After enough of an application is built for a tester to do things and provide useful feedback
+* To test out different possible options for a design
+
+## Schedule
+* A github ticket can be used to track user testing as part of a sprint
+* Include the tech team on the invite; they do not all have to come
+* Ideally at least 2 team members should attend: one to "guide" the testing, and a "recorder" for note taking
+* When possible, prepare to record the meeting
+  * Also let the tester know we will record the testing session, make sure they are aware and ok with it
+* Use a meeting invite to schedule the tester, and reserve a conf room for team members
+  * Include context and expectation setting for the tester (todo: template email)
+* How long to schedule depends on how much will be tested, usually 0.5 - 1 hour
+* Send out a link and instructions on how the tester and team can get into the session
+  * Even if the tester is in person, use zoom/slack/hangouts so remote team members can join and observe
+  * Include info about screen sharing for the participant
+* Also schedule a debrief meeting after the testing session to discuss findings with team members
+  * Ideally, immediately following testing, or ASAP
+
+## Preparation
+* Make sure any login credentials are ready, and tested to make sure the tester can get into the application
+* Check any data is setup and makes sense, ideally use data the tester will find meaningful
+* Have team member roles assigned
+* Make sure the A/V setup is working
```

Comment:
We have also been preparing the questions and user activities/scenarios in advance

Reply:
oh, good one! I knew I was forgetting important things